test/check-machines-create

```diff
@@ -139,6 +140,49 @@ class TestMachinesCreate(VirtualMachinesCase):
                                                                          create_and_run=False),
                                              {"source-url": "Installation source must not be empty"})
 
+    def testCreateNameGeneration(self):
```

Comment:
Please add this to the exclude list in test/browser/run-test.sh , as it tests internal logic only.